### PR TITLE
Update spix to latest master revision to gain ability to invoke QML functions

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 448c9a3f0e1b26be9693f0a1dc1fb4e34a6c60e4: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3366-cdf9fb247772104c42874d8abb8cfcefce058377-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3366-cdf9fb247772104c42874d8abb8cfcefce058377-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3366-cdf9fb247772104c42874d8abb8cfcefce058377-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3366-cdf9fb247772104c42874d8abb8cfcefce058377-x86-android.apk)
</details>

File: vcpkg/overlay/spix/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    SHA512 07555c17f1836e1297ff91cc5dd05f7171a2a23dc6896be4bb6ab88b9b8d96ff0908b41758b4effcc719257a9b7eacff258a7fece896d5075082afcbfe534fc0
+    HEAD_REF master
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DSPIX_BUILD_EXAMPLES=OFF
+        -DSPIX_BUILD_TESTS=OFF
+        -DSPIX_QT_MAJOR=5
```
Comment (m-kuhn) [inline vcpkg/overlay/spix/portfile.cmake:14]: Also to be adjusted in spix-qt6?

Comment (nirvn) [reply vcpkg/overlay/spix/portfile.cmake:14]: Fair point sir.

**Comment by nirvn** [comment]
@m-kuhn , spix-qt6 updated

File: vcpkg/overlay/spix/vcpkg.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "spix",
+  "version": "0.4",
```
Comment (m-kuhn) [inline vcpkg/overlay/spix/vcpkg.json:3]:
```suggestion
  "version": "dev",
```

**Review by m-kuhn — APPROVED** [review]

File: vcpkg/overlay/spix/vcpkg.json (hunk 2 above)
Comment (nirvn) [reply vcpkg/overlay/spix/vcpkg.json]: TIL: "version" vs. "version-string" :)

**Pull request merged**